circuit-codegen-annotations/build.gradle.kts

```diff
@@ -13,13 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import org.jetbrains.kotlin.gradle.plugin.PLUGIN_CLASSPATH_CONFIGURATION_NAME
+
 plugins {
-  kotlin("jvm")
+  id("com.android.library")
+  kotlin("multiplatform")
   alias(libs.plugins.mavenPublish)
 }
 
-dependencies {
-  api(libs.anvil.annotations)
-  api(libs.dagger)
-  api(projects.circuit)
+kotlin {
+  // region KMP Targets
+  android { publishLibraryVariants("release") }
+  jvm()
+  // endregion
+
+  sourceSets {
```

Comment:
for my understanding, what is the `maybeCreate` doing here?

Reply:
It creates the source set if it doesn't exist, returns the existing one if it does